plugins/anomali_threatstream/unit_test/test_get_sandbox_report.py

```diff
@@ -0,0 +1,72 @@
+import sys
+import os
+
+sys.path.append(os.path.abspath("../"))
+
+from unittest import TestCase
+from komand_anomali_threatstream.connection.connection import Connection
+from komand_anomali_threatstream.actions.get_sandbox_report import GetSandboxReport
+import json
+import logging
+
+
+class TestGetSandboxReport(TestCase):
```

Comment:
You need to remove all the integration tests throughout the plugin. These are passing in a very cheaty way. There's no __init__.py which means they aren't found or run.